modules/running-distributed-data-science-workloads-disconnected-env.adoc

```diff
@@ -22,6 +23,21 @@ To run a distributed data science workload in a disconnected environment, you mu
 . In the `ClusterConfiguration` section of the notebook or pipeline, ensure that the `image` value specifies a Ray cluster image that can be accessed from the disconnected environment:
 * Notebooks use the Ray cluster image to create a Ray cluster when running the notebook.
 * Pipelines use the Ray cluster image to create a Ray cluster during the pipeline run.
+. In the CodeFlare Operator config map, ensure that the `kuberay:certGeneratorImage` value specifies an image that contains the `openssl` package, and that the image can be accessed from the disconnected environment.
```

Comment:
"can be accessed" or "is accessible" ?

Reply:
Thanks, @chtyler - rephrased to avoid passive tense.